document_page_environmental_aspect/data/document_page.xml

```diff
@@ -1,26 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<odoo>
-    <record id="document_page_group_environmental_aspect" model="document.page">
-      <field name="name">Environmental Aspect</field>
-      <field name="type">category</field>
-      <field name="template">
-        <![CDATA[
-<h1>Activity</h1>
+<odoo noupdate="1">
 
-<h1>Environmental Aspect</h1>
-
-<h1>Environmental Impact</h1>
-
-<h1>Laws and Regulations</h1>
-
-<h1>Operational Control</h1>
-
-<h1>Responsibilities</h1>
-
-<h1>Objectives</h1>
-
-<h1>Programs</h1>
-        ]]>
-      </field>
+    <record id="environment_aspect_tag" model="document.page.tag" forcecreate="1">
+        <field name="name">Environment Aspect</field>
```

Comment:
@etobella I would keep the original as it was and add a new one `document_page_tag.xml` with this content.